gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/DefaultHttpAcceptor.java

```diff
@@ -16,9 +16,15 @@
 package io.gravitee.gateway.reactor.handler;
 
 import io.gravitee.gateway.api.Request;
+import java.util.Objects;
 import java.util.regex.Pattern;
 
 /**
+ * Comparator used to sort {@link HttpAcceptor} in a centralized acceptor collection.
+ * The final entrypoint collection is a {@link java.util.concurrent.ConcurrentSkipListSet} which rely on this comparator to add / remove entry keeping entries ordered.
```

Comment:
I think it is not relevant anymore as we remove the use of `ConcurrentSkipListSet` since it caused a lot of problems